database/seeds/init.js

```diff
@@ -0,0 +1,61 @@
+const fs = require('fs');
+
+const populateImageJoin = (start, end) => {
+  const result = [];
+  for (let productId = start; productId < end; productId += 1) {
+    let rows = [];
+    for (let positionIndex = 0; positionIndex < 5; positionIndex += 1) {
+      const randomImageId = Math.floor(Math.random() * 1000);
+      rows.push([productId, randomImageId, positionIndex].join(','));
+    }
+    rows = rows.join('\n');
+    result.push(rows);
+  }
+  return result.join('\n') + '\n';
+};
+
+const populateImages = () => {
+  let imagesDoc = [];
+  for (let i = 0; i < 1000; i++) {
+    imagesDoc.push([i, `https://s3-us-west-1.amazonaws.com/imagesrapidretail/${i}.jpg`].join(','));
+  }
+  imagesDoc = imagesDoc.join('\n');
+  fs.appendFileSync('./images_seed.csv', imagesDoc);
+};
+
+const genders = ['Men', 'Women', 'Girls'];
+const categories = ['Tops', 'Bottoms', 'Swim'];
+const type = ['Pants', 'Shorts', 'Shirts'];
+
+const populateProductsData = (start, end) => {
+  const result = [];
+  for (let i = start; i < end; i += 1) {
+    result.push([
+      i,
+      genders[Math.floor(Math.random() * 3)],
+      categories[Math.floor(Math.random() * 3)],
+      type[Math.floor(Math.random() * 3)],
+    ].join(','));
+  }
+  return result.join('\n') + '\n';
+};
+
+populateImages();
+
+let start = 1;
+let end = 100001;
+
+while (start < 10000001) {
```

Comment:
Instead of using a while loop here, why not just use a for loop and write to the file each time i % 100000 === 0?